manifest.txt

```diff
@@ -7,52 +7,73 @@ service-policies/engine:com.fasterxml.jackson.module:jackson-module-parameter-na
 service-policies/engine:com.github.luben:zstd-jni:jar:1.4.4-7
 service-policies/engine:com.google.code.findbugs:jsr305:jar:3.0.2
+service-policies/engine:com.google.code.gson:gson:jar:2.8.6
 service-policies/engine:com.google.errorprone:error_prone_annotations:jar:2.2.0
 service-policies/engine:com.google.guava:failureaccess:jar:1.0.1
 service-policies/engine:com.google.guava:guava:jar:28.1-jre
 service-policies/engine:com.google.guava:listenablefuture:jar:9999.0-empty-to-avoid-conflict-with-guava
 service-policies/engine:com.google.j2objc:j2objc-annotations:jar:1.3
 service-policies/engine:com.google.protobuf:protobuf-java:jar:3.12.2
-service-policies/engine:commons-codec:commons-codec:jar:1.14
+service-policies/engine:com.ibm.async:asyncutil:jar:0.1.0
+service-policies/engine:com.squareup.okhttp3:okhttp:jar:3.14.9
+service-policies/engine:com.squareup.okio:okio:jar:1.17.2
+service-policies/engine:com.sun.activation:jakarta.activation:jar:1.2.1
 service-policies/engine:com.thoughtworks.xstream:xstream:jar:1.4.11.1
+service-policies/engine:commons-codec:commons-codec:jar:1.14
+service-policies/engine:io.jaegertracing:jaeger-core:jar:0.34.3
+service-policies/engine:io.jaegertracing:jaeger-thrift:jar:0.34.3
 service-policies/engine:io.netty:netty-buffer:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-codec-dns:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-codec-http2:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-codec-http:jar:4.1.49.Final
-service-policies/engine:io.netty:netty-codec:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-codec-socks:jar:4.1.49.Final
+service-policies/engine:io.netty:netty-codec:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-common:jar:4.1.49.Final
-service-policies/engine:io.netty:netty-handler:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-handler-proxy:jar:4.1.49.Final
+service-policies/engine:io.netty:netty-handler:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-resolver-dns:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-resolver:jar:4.1.49.Final
-service-policies/engine:io.netty:netty-transport:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-transport-native-epoll:jar:4.1.49.Final
 service-policies/engine:io.netty:netty-transport-native-unix-common:jar:4.1.49.Final
+service-policies/engine:io.netty:netty-transport:jar:4.1.49.Final
+service-policies/engine:io.opentracing.contrib:opentracing-concurrent:jar:0.2.0
+service-policies/engine:io.opentracing.contrib:opentracing-interceptors:jar:0.0.4
+service-policies/engine:io.opentracing.contrib:opentracing-jaxrs2:jar:0.4.1
+service-policies/engine:io.opentracing.contrib:opentracing-tracerresolver:jar:0.1.8
+service-policies/engine:io.opentracing.contrib:opentracing-vertx-web:jar:1.0.0
+service-policies/engine:io.opentracing.contrib:opentracing-web-servlet-filter:jar:0.2.3
+service-policies/engine:io.opentracing:opentracing-api:jar:0.31.0
```

Comment:
Should probably target 0.33 and not 0.31. 0.33 has been out for a year. Although since OpenTracing has been discontinued, shouldn't we use OpenTelemetry from the start?

Reply:
0.31 is pulled in via SR-OpenTracing.
Afaik, there is no Quarkus-OpenTelemetry available yet

Reply:
I don't think quarkus-opentracing makes any sense here. We don't use anything from it. opentelemetry-java-instrumention has native support for vertx.